docs/API-REFERENCE.md

```diff
@@ -1 +1,2640 @@
-See JerryScript API documentation on [JerryScript home page](https://samsung.github.io/jerryscript/API/).
\ No newline at end of file
+# jerry_run_simple
+
+**Summary**
+
+The simplest way to run JavaScript.
+
+**Prototype**
+
+```c
+jerry_completion_code_t
+jerry_run_simple (const jerry_char_t *script_source,
+                  size_t script_source_size,
+                  jerry_flag_t flags);
+```
+
+- `script_source` - source code, it must be a valid utf8 string.
+- `script_source_size` - size of source code buffer, in bytes.
+- return value - completion code that indicates whether the run was performed successfully
```

Comment:
`run was performed` feels a bit weird to me. Could we say something like `completion code, indicating whether execution was successful or not`?